nixos/modules/profiles

```diff
@@ -0,0 +1 @@
+../profiles/
```

Comment:
Is this a symlink? One thing to note is nixpkgs currently has no symlinks, and it may be worth a bit of effort to keep it that way. I've considered adding a check for this in Ofborg, because symlinks can evade some of the restricted evaluation settings.

Reply:
I can replace the symlinks with stubs that warn + import.